src/nodeTypesStore.js

```diff
@@ -61,9 +62,47 @@ export default new Vuex.Store({
     },
   },
   actions: {
-    // eslint-disable-next-line no-unused-vars
-    getUserPinnedObjects(state) {
-      // here goes the axios call to get the user pinned objects
+    getUserPinnedObjects({ commit }) {
```

Comment:
I just noticed this logic is not taken into consideration for standalone. Meaning, if the API is not available, what will it render?

Reply:
Will work on this ASAP :runner: